src/server/routes/consents.ts

```diff
@@ -558,12 +560,13 @@ router.post(
         url = addReturnUrlToPath(url, state.queryParams.returnUrl);
       }
       return res.redirect(303, url);
-    } catch (e) {
-      logger.error(`${req.method} ${req.originalUrl}  Error`, e);
-      status = e.status;
+    } catch (error) {
+      logger.error(`${req.method} ${req.originalUrl}  Error`, error);
+      const { status: errorStatus, message } = error as RequestError;
+      status = errorStatus;
```

Comment:
Do we do anything with `status` here?

Also, do we need to rename it to `errorStatus` if we are just going to reassign it back to `status` straight away? Sorry if I'm missing some context on that one - possible!

Reply:
yep, so it was engineered so that there's a single `status` variable for the whole request. above this, status is set to `200` and if an error is caught here it is only then set to the `errorStatus` from the error.

below this we then return the status from the controller